frontend/src/Components/Map/MonthFilter.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import DatePicker from 'react-datepicker';
+
+interface Props {
+  /**
+   * date: date input
+   * onStartChange: function taking argument "date" for Start Date
+   * onEndChange: function taking argument "date" for End date
+   */
+
+  onStartChange: (date: any) => void;
+  onEndChange: (date: any) => void;
+}
+
+/**
+ * Component rendering Month Filter
+ */
+
+const MonthFilter: React.FC<Props> = ({ onEndChange, onStartChange }) => {
+  const [startDate, setStartDate] = useState(new Date('2022/06/08'));
+  const [endDate, setEndDate] = useState(new Date());
+
+  return (
+    <>
```

Comment:
Show ui bug 
![datafilter](https://github.com/DTU-SE-Group-D/LiRA-Viz/assets/65665540/9f1e108d-49f5-4f9f-b943-475f42c7abfb)

Reply:
Fixed by adding a `div` around each component and css (`display: inline-block`) for these `div`s